opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -40,23 +45,19 @@
 _RANDOM_ID_GENERATOR = "random"
```

Comment:
I believe `_EXPORTER_OTLP_SPAN` can be renamed to `_EXPORTER_OTLP_DEFAULT` or something now. ^

Reply:
renamed to _EXPORTER_OTLP_PROTO_GRPC